### PR TITLE
Create assets and update README with examples

### LINKED ISSUE #19: Add example profile to README and/or docs
We could show what the package produces with a short example accompanied by a profile.

### PR DESCRIPTION
**Pull request by abelsiqueira**
Closes #19 

- Created folder `assets` with a focused script using pyplot;
- still needs an updated documentation;
- I'm having problems with the `profile2` image, it's cropping the right side.

### CODE REVIEW

File: assets/generate_assets.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  # Alpha table
+  df = stats[:alpha]
+  cols = [:status, :name, :f, :t, :iter]
+  header = Dict(:status => "flag", :f => "\\(f(x)\\)", :t => "time")
+  open("alpha.tex", "w") do io
+    latex_table(io, df, cols=cols, hdr_override=header)
+  end
+  run(`latexmk -pdf alpha-standalone.tex`)
+  run(`convert -density 150 alpha-standalone.pdf -background white -flatten alpha.png`)
```
Comment (dpo) [inline assets/generate_assets.jl]: Could we make it an `svg`?

Comment (abelsiqueira) [reply assets/generate_assets.jl]: I couldn't find an easy way to do it. Tried with ImageMagick and `gs`.

Comment (dpo) [reply assets/generate_assets.jl]: At least on macOs, I have `pdf2svg`, but no big deal.

**Comment by dpo** [comment]
Thank you!

**Pull request merged**